components/scream/src/share/scream_pack_kokkos.hpp

```diff
@@ -254,6 +254,51 @@ smallize (const Kokkos::View<BigPack<T>*, Parms...>& vp) {
   return repack<SCREAM_SMALL_PACK_SIZE>(vp);
 }
 
+//
+// Take an array of Host scalar pointers and turn them into device pack views
+//
+template <Int N, typename Scalar, typename PackT, typename DeviceT>
```

Comment:
I found that I needed functionality "Take host raw scalar data, move into device pack view" frequently in the _f functions. This was my attempt to encapsulate this capability for reuse. I'm currently a bit disappointed in how it turned out for a couple reasons:
1) The compiler doesn't seem to be able to infer the template arguments at all, so all 4 have to be provided.
2) Kokkos::Array<T, N> arguments don't seem to be able to be used with a Kokkos::Array<const T, N> signature

Despite these drawbacks, as you can see above in p3_functions_f90.cpp, it saves us a ton of code.

Reply:
Also, it occurs to me that, if I'm going to offer this capability in scream_pack_kokkos, I should probably add a unit test.